app/controllers/register_controller.rb

```diff
@@ -0,0 +1,25 @@
+class RegisterController < ApplicationController
+
+  def new
+  end
+
+  def create
+    @user = User.new(user_params)
+    if User.exists?(email: user_params[:email])
+      flash[:error] = "Email already in use. Try a different email."
+      render :new
+    elsif @user.save
```

Comment:
we should later on add a statement that checks if confirm password is the same as password